packages/handlers/grpc/src/index.ts

```diff
@@ -512,8 +526,20 @@ ${rootJsonEntries
 
   private schemaComposer = new SchemaComposer();
 
-  async getMeshSource() {
+  async getMeshSource({ fetchFn }: GetMeshSourcePayload): Promise<MeshSource> {
     this.config.requestTimeout = this.config.requestTimeout || 200000;
+    this.fetchFn = fetchFn;
+
+    if (typeof this.config.source === 'string') {
+      const interpolatedSource = stringInterpolator.parse(this.config.source, {
+        env: process.env,
+      });
+      if (interpolatedSource.endsWith('.graphql')) {
+        return {
+          schema: await this.getExistingGraphqlSchema(interpolatedSource),
```

Comment:
unfortunately this won't work because it needs the execution layer to be applied :)